msb/rawlogger/msb_rawlogger.py

```diff
@@ -32,20 +33,25 @@ def get_data(self) -> tuple[bytes, bytes]:
                 continue
             yield topic, data
 
-    def run(self):
-        for topic, data_raw in self.get_data():
-            if self.config.print_stdout:
-                print(f"{topic.decode('utf-8')} : {data_raw.decode('utf-8')}")
 
-            data = b'{"' + topic + b'":' + data_raw + b"}\n"
-            self.logger.write(data)
+    def run(self,excluded_topics):
+        for topic, data_raw in self.get_data(excluded_topics):
+            if topic.decode('utf-8') in excluded_topics:
+                pass
+            elif self.config.print_stdout:
```

Comment:
This changes the behavior of the print_stdout flag. This should print additionally to logging, not instead.

Reply:
New Code version should have fixed this issue